### DISCUSSION TITLE
Update the Dataview integration default case, configuration and documentation

### DISCUSSION THREAD

**Discussion by mathisgauthey** [davish/obsidian-full-calendar#519] [Feature Requests]
### ⚠️ Please check that this feature request hasn't been suggested before.

- [X] I searched previous [Ideas in Discussions](https://github.com/davish/obsidian-full-calendar/discussions/categories/feature-requests) didn't find any similar feature requests.
- [X] I searched previous [Issues](https://github.com/davish/obsidian-full-calendar/issues) didn't find any similar feature requests.

### 🔖 Feature description

I think that the Dataview integration is an important feature and this plugin could benefit from it being simpler.

For instance, my vault includes a daily note template with tasks queries. My full-calendar includes .ics calendar, my google calendar for events and time-blocking.

Right now I'm using [obsidian-google-calendar](https://github.com/YukiGasai/obsidian-google-calendar), but the design is painful.
- On desktop I can use the snippet of the web-view which does the job but breaks with obsidian design.
- On mobile, I need to use the included design that I find lacking, especially in comparison to the way `obsidian-full-calendar` renders, which I find better.

### ✔️ Solution

My solution would be to update the [documentation](https://davish.github.io/obsidian-full-calendar/advanced/dataview/) with a standard case that import exactly the same full-calendar you get by using the command `Full Calendar : Open Calendar`.

It should provide some ways to define a basic view (month, week, day, list) for desktop and one for mobile (3 days, day, list).

And you should be able to select your sources easily on the snipped if you want to use only external or internal calendar, for instance.

### 📝 Additional Context

I saw that this is quite a demanded feature, with some discussions about it : 
- [Render main calendar in-line with a note with Dataview #362](https://github.com/davish/obsidian-full-calendar/discussions/362)
- [Access remote events from dataviewjs #394](https://github.com/davish/obsidian-full-calendar/discussions/394)

And there's two pull-request : 
- [ Default Calendar Sources for renderCalendar + Cache Bugfix (Breaking Option) #456 ](https://github.com/davish/obsidian-full-calendar/pull/456)
- [ Default Calendar Sources for renderCalendar + Cache Bugfix (NonBreaking Option) #457 ](https://github.com/davish/obsidian-full-calendar/pull/457)

**Comment by ruben-sg** [comment]
Any update on this?
